entrypoint.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+set -e
+
+git submodule init
+git submodule update
+
+echo "#################################################"
```

Comment:
It looks like you are removing the readme and .gitignore. Is that for the published site or on my code repo? If it is on my GitHub code repo, I do not want to delete those.

Reply:
The files aren't removed on any repository. If I don't remove them, they are pushed to the target repository, where they aren't necessary.